test/functional.js

```diff
@@ -1,15 +1,16 @@
 const assert = require('assert');
 const got = require('got');
+const parallel = require('mocha.parallel');
 
-describe('functional', () => {
+parallel('functional', () => {
   let server;
 
-  before((done) => {
+  before((done) => { // eslint-disable-line mocha/no-top-level-hooks
```

Comment:
I raised an issue for this problem https://github.com/lo1tuma/eslint-plugin-mocha/issues/125